database/migrations/2025_04_12_000000_update_gamedata_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('GameData', function (Blueprint $table) {
+            $table->dateTime('achievements_published_at')->nullable()->default(null)->after('RichPresencePatch');
```

Comment:
This field could be used for https://discord.com/channels/310192285306454017/1362385885513126150/1362385885513126150, but doesn't help with revisions.